Hermes/appscale/hermes/helper.py

```diff
@@ -309,6 +308,7 @@ def backup_apps(storage, bucket):
   Returns:
     True on success, False otherwise.
   """
+  from appscale.datastore.backup import backup_recovery_helper as BR
```

Comment:
[PEP8#imports](https://www.python.org/dev/peps/pep-0008/#imports) says:

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

But in this case we always save ~10MB of resident memory for all slave nodes and in most cases we save the same amount of memory on master node.
[Discussion on stackoverflow](http://stackoverflow.com/questions/128478/should-python-import-statements-always-be-at-the-top-of-a-module) with a high rating considers such solution as acceptable.